src/app/utils/Comparable.ts

```diff
@@ -3,7 +3,8 @@ import { JSONPrimitive } from './utils';
 export interface ComparableObject {
     equals(o: ComparableObject): boolean;
 
-    toString(): string;
+    // Removed because: why would a comparable need a toString, this seems totally unrelated?
+    // toString(): string;
```

Comment:
More for debugging purpose I guess ?
If it's to remove then we must do a ticket for it so we remove "dead code" as well, but I'd say it's good to have that